app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
```

Comment:
Why did you decide to use a new Dashboard controller instead of `UserController#show`? They're showing the same information.